src/goto-instrument/nondet_volatile.cpp

```diff
@@ -101,27 +108,67 @@ bool nondet_volatilet::is_volatile(const namespacet &ns, const typet &src)
 
 void nondet_volatilet::handle_volatile_expression(
   exprt &expr,
-  const namespacet &ns)
+  const namespacet &ns,
```

Comment:
:pick: bd81144c0d2fb0347497adf3fa1df20b021a4cc7

Says it adds a new `--nondet-volatile-model` option, but this option is actually added in a different commit.

Reply:
Changed commit message.